### DISCUSSION TITLE
Better way to highlight what is capabilities and what is outcomes

### DISCUSSION THREAD

**Discussion by Winael** [dora-team/dora.dev#359] [Ideas]
Hi Everyone,

For now, it's quite difficult in the ressource to differentiate outcomes from capabilities. For Example, Well being category and Job Satisfaction are detailed in `./devops-capabilites/cultural/` folder and list under Capabilities in the website, when SDO and Organizational Performance are in `./concepts` folder.

Could it be possible to all the outcomes indentified as it, and maybe in the same hierarchy?

Regards,

Vincent JOBARD

**Comment by davidstanke** [comment]
Thanks for the feedback! There's room for debate but we've tried to assert a clear distinction between these types of ideas in the [core model](https://dora.dev/research). However, the site's content hierarchy doesn't necessarily follow that. It would, indeed, be good to better align these things.
